edge-modules/ModuleLib/ModuleUtil.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace ModuleLib
+{
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Client.Transport.Mqtt;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Edge.Util.TransientFaultHandling;
+    using Microsoft.Extensions.Logging;
+    using Serilog;
+    using Serilog.Core;
+    using Serilog.Events;
+    using ExponentialBackoff = Microsoft.Azure.Devices.Edge.Util.TransientFaultHandling.ExponentialBackoff;
+    using ILogger = Microsoft.Extensions.Logging.ILogger;
+
+    public static class ModuleUtil
+    {
+        public static readonly ITransientErrorDetectionStrategy DefaultTimeoutErrorDetectionStrategy =
+            new DelegateErrorDetectionStrategy(ex => ex.HasTimeoutException());
+
+        public static readonly RetryStrategy DefaultTransientRetryStrategy =
+            new ExponentialBackoff(
+                5,
+                TimeSpan.FromSeconds(2),
+                TimeSpan.FromSeconds(60),
+                TimeSpan.FromSeconds(4));
+
+        public static async Task<ModuleClient> CreateModuleClientAsync(
+            TransportType transportType,
```

Comment:
It feels like it might be better to do this in a base class instead of in util class..

Reply:
discussed, just keep as is for now.